src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValue.cs

```diff
@@ -69,13 +69,14 @@ public IList<NameValueHeaderValue> Parameters
 
         public StringSegment Name
         {
-            get { return GetName(NameString); }
+            get { return HeaderUtilities.RemoveQuotes(GetName(NameString)); }
```

Comment:
How often will the getter be called @Tratcher ?We can store a local copy after RemoveQuotes has been called once, but it would cause a bit of clutter because only Name and FileName would have these local copies.